project_forecast_line/models/forecast_line.py

```diff
@@ -140,34 +143,55 @@ def _compute_employee_forecast_line_id(self):
             else:
                 rec.employee_resource_forecast_line_id = False
 
-    def _convert_forecast(self, data):
-        """
-        Converts consolidated forecast from hours to days
-        """
-        self.ensure_one()
+    def _get_grouped_line_values(self):
+        data = {}
+        grouped_line_result = self.env["forecast.line"].read_group(
+            [("employee_resource_forecast_line_id", "in", self.ids)],
+            fields=["forecast_hours"],
+            groupby=["employee_resource_forecast_line_id", "type"],
+            lazy=False,
+        )
+        for d in grouped_line_result:
+            line_id = d["employee_resource_forecast_line_id"][0]
+            if line_id not in data:
+                data[line_id] = {}
+            data[line_id][d["type"]] = d["forecast_hours"]
```

Comment:
```suggestion
                data[line_id] = {'confirmed': 0, 'forecast': 0}
            data[line_id][d["type"]] += d["forecast_hours"]
```

Reply:
I think you need the sum, please double check what you get if you have 2 confirmed tasks on the same day.

Reply:
@gurneyalex I've updated the code and tests. Thanks, for the review.
Though Travis is failing because of the `project_task_code` unique sequence.
Use of `@tagged("-at_install", "post_install")` doesn't help. Any ideas?